Assets/Src/AuthController/REST/HttpPortListener.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading;
+using UnityEngine;
+
+namespace Src.AuthController.REST
+{
+    public class HttpPortListener
+    {
+        private readonly HttpListener _listener;
+        private Thread _listenerThread;
+        private Action<string> _onCodeFetched;
+        
+        private const string ResponseHtml = "<h1>You can return to the app now!</h1>"; // TODO: Change this to a successful html response
+
+        public HttpPortListener(int port)
+        {
+            _listener = new HttpListener();
+            _listener.Prefixes.Add($"http://localhost:{port}/");
+            _listener.Prefixes.Add($"http://127.0.0.1:{port}/");
+            _listener.AuthenticationSchemes = AuthenticationSchemes.Anonymous;
```

Comment:
Class creates HttpListener with localhost prefixes and some AuthentiacationScheme, but it is not clear from the class name. Consider either renaming this class to clarifiy what is exact purpose of it or redesigning it.